analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PInvokesShouldNotBeVisible.CSharp11.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Runtime.InteropServices;
+
+public interface IInterface
+{
+    [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
```

Comment:
Discussed offline: These tests should stay.
There's an [issue](https://github.com/dotnet/runtime/issues/76673) during runtime regarding inheritance in interfaces - it's good to have tests to catch any possible regressions after its fixed (since roslyn might be affected).